### PR TITLE
Refine "GraphEditor" component, extend sample graphs

### PR DESCRIPTION
**Pull request by songofroland**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -54,15 +54,15 @@
   max-width: 100%;
 }
 
-.valid-base {
```
Comment (wilmhit) [inline src/App.css:57]: Is the text still centered?

File: src/components/GraphEditor.tsx [hunk 2]
```diff
@@ -1,66 +1,91 @@
...
-    </div>
-  </div>;
+    </div>;
+  </>
 }
 
-function GraphChooser({ possibleValues, selectCallback }
-  : { possibleValues: any[], selectCallback: Function })
+function GraphChooser({ possibleValues, selectCallback, selectedLabel }
+  : { possibleValues: any[], selectCallback: Function , selectedLabel: string})
```
Comment (wilmhit) [inline src/components/GraphEditor.tsx:72]: We were using `Array<any` notation up to this point so lets stick with that

File: src/logic/graphSamples.ts [hunk 3]
```diff
@@ -1,11 +1,32 @@
...
+    string: '0 2 6 0 0 0 0\n2 0 0 5 0 0 0\n6 0 0 8 0 0 0\n0 5 8 0 10 15 0\n0 0 0 10 0 6 2\n0 0 0 15 6 0 6\n0 0 0 0 2 6 0',
+  },
   simple: {
-    graph: [[0, 1, 1], [1, 0, 1], [1, 1, 0]],
-    string: '0 1 1\n1 0 1\n1 1 0',
+    graph: [
+      [0, 1, 1, 1], 
+      [1, 0, 1, 1], 
+      [1, 1, 0, 1], 
+      [1,1,1,0],
```
Comment (wilmhit) [inline src/logic/graphSamples.ts:20]: If you're still using vim keybindings press == while in normal mode in this line.

File: src/components/GraphEditor.tsx [hunk 4]
```diff
@@ -1,66 +1,91 @@
...
 
 
 function GraphEditor({ onRender }: { onRender: Function }) {
-  const [userInput, setUserInput] = useState({
-    isCorrect: true,
-    text: samples.simple.string,
-  });
+  const [inputText, setInputText] = useState(samples.simple.string);
+  const [inputTextIsCorrect, setInputTextIsCorrect] = useState(true);
+  const [selectedGraphName, setSelectedGraphName] = useState('simple');
```
Comment (wilmhit) [inline src/components/GraphEditor.tsx:25]: `Name` resembles more `label` field rather than `key` field

**Review by wilmhit — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (songofroland) [reply src/App.css:57]: It wasn't, good catch

**Pull request merged**